tools/config.js

```diff
@@ -63,9 +63,7 @@ Promise.resolve(userToken || cachedToken)
         }
       ]);
 
-      const {
-        projectId: { id }
-      } = response;
+      const { projectId: { id } } = response;
```

Comment:
You probably haven't run "yarn" from the root of the repo recently and so you're using the wrong version of the formatter.  @jshcrowthe We really should have a version check in the pre-push hook to prevent this. :-)

Reply:
Got it in one, thanks @mikelehen!